surfactant/filetypeid/id_magic.py

```diff
@@ -162,6 +162,18 @@ def identify_file_type(filepath: str) -> Optional[str]:
                 int.from_bytes(magic_bytes[0:4], byteorder="big", signed=False) & 0xFF0F80FF
             ) == 0xF00D0000:
                 return "OMF_LIB"
+            # Check for script files
+            if magic_bytes[:12] == b"#!/bin/bash\n" or suffix == ".sh":
+                return "BASH"
+            if suffix == ".py":
```

Comment:
I think `#!/bin/python` is one of the things that can appear at the top of some Python files without a `.py` extension -- check the GE G500 python scripts to confirm that's what it uses (or ask Alex Armstrong).

I think we can also add a check for a `.pyc` extension, that returns something like `PYTHON_COMPILED` or `PYTHON_BYTECODE`?